contracts/contracts/liquidity/LiquidityReward.sol

```diff
@@ -275,18 +272,20 @@ contract LiquidityReward is Initializable, Governable {
         UserInfo storage user = userInfo[msg.sender];
         updatePool();
         if (_amount > 0) {
+            user.amount = user.amount.add(_amount);
+            // newDebt is equal to the change in amount * accRewardPerShare (note accRewardPerShare is historic)
+            int256 newDebt = int256(
+                _amount.mulTruncate(pool.accRewardPerShare)
+            );
+            user.rewardDebt += newDebt;
```

Comment:
It's unlikely we would ever get an overflow, but for code consistency purposes, should safeMath be used for operations on debt as well?
```
user.rewardDebt.add(newDebt)
totalRewardDebt.add(newDebt)
```

Reply:
rewardDebt can go negative because we allow users to withdraw without claiming the reward. In that case we owe the user a balance.